Khan/wonder-blocks
# Remove Clickable from Choice

## Summary:
Currently, there is an accessibility issue where screenreaders
are reading the wrong checked state for checkboxes and radio
buttons on iOS.

It seems like the cause of this issue has to do with
ClickableBehavior. We can fix this by removing ClickableBehavior
from choice-js.

- Remove ClickableBehavior from choice.js
- Make the styles use pseudo selectors instead of relying on the
  clickable states

Issue: https://khanacademy.atlassian.net/browse/WB-1427


## Test plan:
`yarn jest`

Manual testing
- Go to http://localhost:58796/?path=/story/form-checkbox--variants-controlled
- Hover on the unchecked checkboxes and confirm it looks the same as before.
- Unhover on the checked checkboxes and confirm it looks the same as before.
- Press down on the unchecked checkboxes and confirm it looks the same as before.
- Unpress and confirm that the checked state changes, and it the newly checked checkbox look the same as before.
- Repleat steps for checked checkboxes.
- Turn on a screen reader and confirm that the checked states are all correct.

- Repeat the above steps for radio buttons at http://localhost:58796/?path=/story/form-radiogroup--basic

- Repeat the above steps for checkboxes and radio buttons on storybook on a mobile device.

## Mobile screenreader demos (sound on to hear screen reader)

### Checkbox Before

https://user-images.githubusercontent.com/13231763/221702662-0bc6a99a-84ce-4a27-a281-97cad2134888.mov

### Checkbox After

https://user-images.githubusercontent.com/13231763/221702718-bed8bbf6-9a07-4e7e-a8a3-7b9ecfb7d16a.mov

### Radio Before

https://user-images.githubusercontent.com/13231763/221702683-14ae6137-482f-4ad8-9556-703133ad84b7.MP4

### Radio After

https://user-images.githubusercontent.com/13231763/221702820-61b105ab-e736-447f-bd64-f1abaac92266.mov
